tutorials/intro-express/1-welcome.qmd

```diff
@@ -0,0 +1,310 @@
+---
+title: Getting Started
+---
+
+Shiny for Python is a web application framework that helps tell your
+data story.
+If you've landed on this page,
+you probably have a bit of Python experience,
+worked with data,
+and now need a way to publish an interactive
+web application to help tell your data story.
```

Comment:
I like this style (semantic breaks rather than sentence breaks), but in practice I've found it easier to run these files through the Quarto visual editor to get the canonical pandoc formatting with _sentence breaks_. That ends up being a good middle ground and I like having auto-formatters be the arbiters of style.